images/kafka/config/template.apko.yaml

```diff
@@ -30,6 +30,11 @@ paths:
     permissions: 0o755
     uid: 65532
     gid: 65532
+  - path: /bitnami/kafka
+    type: directory
+    permissions: 0o755
+    uid: 65532
+    gid: 65532
```

Comment:
Do we know what is written here by the compose stuff?  This doesn't appear necessary for the Helm side of things.

Reply:
Hmm, the compose yaml has:
```
    volumes:
      - "kafka_data:/bitnami"
```

I'm curious if adjusting that would change anything 🤔

Reply:
I just updated the volume dir instead of adding another permissions for file, it seems working, thanks.

Reply:
awesome!